pkg/daemon/update.go

```diff
@@ -782,12 +782,35 @@ func (dn *Daemon) deleteStaleData(oldConfig, newConfig *mcfgv1.MachineConfig) er
 
 	for _, f := range oldConfig.Spec.Config.Storage.Files {
 		if _, ok := newFileSet[f.Path]; !ok {
-			if _, err := os.Stat(origFileName(f.Path)); err == nil {
-				if err := os.Rename(origFileName(f.Path), f.Path); err != nil {
-					return err
+			if _, err := os.Stat(noOrigFileName(f.Path)); err == nil {
+				if err := os.Remove(noOrigFileName(f.Path)); err != nil {
+					return errors.Wrapf(err, "deleting no orig file %q: %v", noOrigFileName(f.Path), err)
```

Comment:
this error message seems confusing to a user

Reply:
To be clear I'm having trouble understanding what this noOrigFile.... stuff is doing.

Reply:
`noOrig` just means that there was no original file on disk when we tried to write a file via the MCD. This tells the MCD to not revert a file when the last instance of it is deleted in MachineConfigs.

The reasoning is added to `createOrigFile` function. It's also in the commit message. Where else would you like me to add some comments? Or how else would you like it to be named

Reply:
maybe noOrigFileFlag or noOrigFileStamp?